### DISCUSSION TITLE
Temas de TCC relacionados ao front-end

### DISCUSSION THREAD

**Discussion by ricardogouveia3** [frontendbr/forum#2096] [Discussão]
Olá pessoas.

Estou fazendo uma pós graduação em Tecnologias e Inovações Web, e queria levar meu TCC pro lado do front-end. 

Minha primeira idéia de tema foi "_Diferenças na detecção de mudanças no DOM (Document Object Model) entre frameworks modernos para construção de aplicações web_". Porém (acho que por ser muito técnico) não achei muito conteúdo acadêmico sobre isso, o que poderia levar meu trabalho a ter um desempenho ruim em referencial. 

Estou em busca de um novo tema. Se vocês tem idéias ou já fizeram trabalhos relacionados que possam ser usados para referenciar um trabalho relacionado com a área, poderiam compartilhar?

Pela primeira vez é sério: é pro meu TCC

**Comment by lucalves** [comment]
Olá, vou incluir algumas "sugestões" que me vieram em mente.

- A modernização e a evolução da web nos últimos anos.
- Como a acessibilidade em websites e aplicativos podem ser úteis a "globalização digital".
- Desenvolvimento de um sistema web voltados a pessoas com autismo

Aqui não são temas, porém são abordagens que podem ser interessantes:

- Modernizar um sistema utilizado atualmente na sua universidade, com as tecnologias mais atuais e documentar esse processo realizando a comparação do anterior com o que foi criado.
- Conversar com um professor (ou seu orientador) e pedir orientação. Provavelmente ele possui várias ideias legais como tema pra TCC, só não encontrou alguém disposto a colocar isso em prática.
- Buscar dentro da sua universidade, os últimos artigos publicados, e buscar se inspirar neles e criar algo maior, como por exemplo, abordando um tema parecido mas de forma mais sucinta.

Esses pontos acima me ajudaram bastante na época que tava prestes a fazer meu TCC. 

Espero que tenha te ajudado. 😉
